IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MechanicalFastenerTypeRailJoint/DocPropertySet.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MechanicalFastenerTypeRailJoint" UniqueId="8a5754e7-b440-2d56-380a-78290fa513ee" ApplicableType="IfcMechanicalFastener/RAILJOINT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="SleeperArrangement_24pyO2nn5EOpil0HhjmwF" />
```

Comment:
Property of a fastener?

Reply:
Yes. It defines, if the sleeper in on or between sleeper.